README.md

```diff
@@ -1,25 +1,53 @@
 # Teresa
 
-Teresa is an extremely simple platform as a service that runs on top of Kubernetes.  
-To have a full Teresa setup you'll need the API running as a POD on Kubernetes and the CLI.  
+Teresa is an extremely simple platform as a service that runs on top of [Kubernetes](https://github.com/kubernetes/kubernetes).  
+The API needs a database backend and access to Amazon S3 for storage.  
+To have a full Teresa setup you'll need the API running as a POD and the [CLI](https://github.com/luizalabs/teresa-cli).  
 
 ## API
 
-First create a docker image, for example:
+### Database Backends
 
-```
-make build
-docker tag image_id login/teresa:latest
-docker push login/teresa:latest
-```
+#### SQLite
 
-Create a Kubernetes deployment and expose it as a service:
+This is the default. No configuration needed: a database file `teresa.sqlite` is automatically used.
 
-```
-kubectl create namespace teresa
-kubectl create -n teresa -f teresa.yml
-kubectl expose deployment teresa -n teresa --type=LoadBalancer --port=80 --target-port=8080
-```
+#### MySQL
+
+A non-empty TERESADB_HOSTNAME environment variable automatically selects this backend. The
+other variables are:
+
+* TERESADB_PORT
+* TERESADB_USERNAME
+* TERESADB_PASSWORD
+* TERESADB_DATABASE
+
+### Creating Users
+
+For now you have to manually insert a row into the `users` table, for example on MySQL:
+
+    insert into users
+    (created_at, updated_at, name, email, password, is_admin)
+    values
+    (NOW(), NOW(), "myuser", "myuser@mydomain.com", "hashed_password", 0);
+
+To generate a bcrypt hashed password:
+
+    $ python3 -c 'import bcrypt; print(bcrypt.hashpw("mypassword".encode(), bcrypt.gensalt()))'
```

Comment:
I made an API to create "bcrypt" hashes. After I publish it we can change this tip to use a simple `curl` (`bcrypt` isn't a python3 builtin package).